app/controllers/cinema_halls_controller.rb

```diff
@@ -20,7 +20,7 @@ def create
     @cinema_hall = CinemaHall.new(cinema_hall_params)
 
     if @cinema_hall.save
-      render json: @cinema_hall, status: :created, location: @cinema_hall
+      render json: cinema_hall_hash(@cinema_hall), status: :created, location: @cinema_hall
```

Comment:
why you need `location`?

Reply:
Digged into this and I conclude that in this case I do not need location.

Reply:
👍